test-it/karma/publicapi/TypeWaveform.Test.js

```diff
@@ -40,9 +41,12 @@ describe('Peek/Poke different datatypes', function () {
                 fraction: '0'
             },
             dt: 0,
-            Y: [] // eslint-disable-line id-length
+            Y: [], // eslint-disable-line id-length
+            attributes: {_data: null, _attributes: null}
         });
 
+        /*
```

Comment:
Instead of commenting out the source could we either move it to its own it statement that is marked `xit` or just delete it and leave a comment saying that writeJSON for waveforms is not supported near the top of the file.

Reply:
Done.